### ISSUE TITLE
UX 

### ISSUE THREAD

**Issue by twgonzalez** [uber/nebula.gl#262]
#### Feature request
Add instructions on the exact UI/UX needed to add/remove/edit GeoJson Features.  

#### Background 
I am not sure if the current release has bugs or if the intended UX is just unclear.   But trying to add/remove/edit polygons or line strings doesn't seem to work correctly.   Often times a Polygon can't be closed with double clicking.   Grabbing vertexes is also problematic.  I am not sure if the user is supposed to press keys while clicking/dragging etc.

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Doc update
  - [ ] Test

**Comment by supersonicclay** [comment]
Are you referring to the example application? nebula.gl has different "modes" that keep it flexible as different applications have very different needs (some only allow polygons, some want to use different key presses to change to different modes). That said, we do have documentation of the various modes that nebula supports [here](https://github.com/uber/nebula.gl/blob/master/docs/api-reference/mode-handlers/overview.md), note that nebula does not handle key presses at this time. Only mouse interaction is supported. Having a full set of instructions for how to do all edits will likely depend on the application using nebula.

Regarding closing a polygon with double-click, can you give more specific reproduction steps? If you are in `drawPolygon` mode, it should work to double-click to close the polygon.

Also, more clarity on how grabbing vertexes being problematic would be helpful. Which mode is are you using and what type of geometry are you editing?

**Comment by twgonzalez** [comment]
Hi Clay,  

First, thanks for all the work you guys have put into this!  It really is a great framework and a huge leap forward for using WebGL.   The performance is amazing as well as the considerations of CPU/GPU usage that most WebGL apps disregard.

I think I figured out what was causing the issue.   Safari!   None of the edit modes seem to work correctly on my version of Safari (12.1.1) on OS X (10.14.5).    When in Draw Polygon/LineString mode clicking does not create  vertex.   I do not see any errors in the console.

But, I tried it in Chrome and everything works as expected.   I also had my own local build running in Chrome that wasn't behaving as expected, but I might have introduced some issues by adding an overlay layer (of a map diagram that I wanted to trace.)

The documentation on the edit modes is pretty clear reading it now, the problem was things not working in Safari and I was thinking I must have missed something.

Once again, thanks.
